test/publish/test_publications_by_author.py

```diff
@@ -420,3 +444,355 @@ def test_no_issn(test_session):
+                    "last_name": "Dimensions",
+                    "orcid": "leland-dimensions",
+                },
+            ]
+        },
+        pubmed_json={
+            "MedlineCitation": {
+                "Article": {
+                    "AuthorList": {
+                        "Author": [
+                            {
+                                "ForeName": "Jane",
+                                "LastName": "Pubmed",
+                                "Identifier": {
+                                    "@Source": "ORCID",
+                                    "#text": "jane-pubmed",
+                                },
+                            },
+                            {
+                                "ForeName": "Mike",
+                                "LastName": "Pubmed",
+                                "Identifier": {
+                                    "@Source": "ORCID",
+                                    "#text": "mike-pubmed",
+                                },
+                            },
+                            {
+                                "ForeName": "Leland",
+                                "LastName": "Pubmed",
+                                "Identifier": {
+                                    "@Source": "ORCID",
+                                    "#text": "leland-pubmed",
+                                },
+                            },
+                        ]
+                    }
+                }
+            }
+        },
+        wos_json={
+            "static_data": {
+                "summary": {
+                    "names": {
+                        "name": [
+                            {"display_name": "Jane Wos", "orcid_id": "jane-wos"},
+                            {"display_name": "Mike Wos", "orcid_id": "mike-wos"},
+                            {"display_name": "Leland Wos", "orcid_id": "leland-wos"},
+                        ]
+                    }
+                }
+            }
+        },
+        crossref_json={
+            "author": [
+                {
+                    "given": "Jane",
+                    "family": "Crossref",
+                    "ORCID": "https://orcid.org/jane-crossref",
+                },
+                {
+                    "given": "Mike",
+                    "family": "Crossref",
+                    "ORCID": "https://orcid.org/mike-crossref",
+                },
+                {
+                    "given": "Leland",
+                    "family": "Crossref",
+                    "ORCID": "https://orcid.org/leland-crossref",
+                },
+            ]
+        },
+        sulpub_json={
+            "author": [
+                {"name": "Sulpub, Jane Elizabeth Lathrop"},
+                {"name": "Sulpub, Mike"},
+                {"name": "Sulpub, Leland DeWitt"},
+            ]
+        },
+    )
+
+    # slowly peel away the platform metadata that's available to confirm that we are
```

Comment:
I'd been trying to figure out a strategy to do something like this in a test, so it's good to see it's possible.